taxi/forms.py

```diff
@@ -0,0 +1,28 @@
+from django import forms
+from django.core.exceptions import ValidationError
+import re
+from django.contrib.auth import get_user_model
+from .models import Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = get_user_model()
+        fields = ["license_number"]
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if not re.match(r"^[A-Z]{3}\d{5}$", license_number):
+            raise ValidationError(
+                "Not allowed format"
+            )
```

Comment:
The regular expression `^[A-Z]{3}\d{5}$` restricts the license number to exactly three uppercase letters followed by five digits. Ensure this format is correct according to the task requirements. If the format should allow for more variations, adjust the regex accordingly. Additionally, the error message 'Not allowed format' could be more descriptive, such as 'License number must be in the format ABC12345'.